po/de.po

```diff
@@ -36494,12 +36494,8 @@ msgid "Run Naev in full-screen mode"
 msgstr "Naev im Vollbildmodus ausführen"
 
 #: src/conf.c:978
-msgid "Use video modesetting when fullscreen is enabled (SDL2-only)"
-msgstr "Video-Moduseinstellung benutzen, falls Vollbild aktiviert ist (nur für SDL2)"
-
-#: src/conf.c:982
-msgid "Minimize on focus loss (SDL2-only)"
-msgstr "Bei Fokusverlust minimieren (nur für SDL2)"
```

Comment:
It's not clear to me why you removed this translation (only the SDL2 part should be removed no?)

Reply:
Hi, it's because the string "Minimize on focus loss" already had a translation in the same po file, for options.c. (The SDL2 disclaimer was only written into conf.lua; the options UI knew it didn't need it.)

Reply:
I see, thanks!